src/Database/RavenCommand.ts

```diff
@@ -86,7 +86,7 @@ export abstract class RavenCommand {
     return options;
   }
 
-  public setResponse(response: IResponse): IRavenCommandResponse | null | void {
+  public setResponse(response: IResponse): IRavenResponse | IRavenResponse[] | null | void {
```

Comment:
i'll research this issue
i tries to define somethign like type RavenResponse = IRavenResponse | IRavenResponse[] | null | void; but in use it requires a lot of "as" casts